src/content/admin/resource-manager.mdx

```diff
@@ -0,0 +1,127 @@
+The Resource Manager automates this process, dynamically adjusting resource requests based on real-time usage data of the Okteto Cluster.
+This ensures your pods are running efficiently and your nodes are balanced well, improving overall performance and developer experience.
+
+### Key Benefits
+- **Improved Workflow Productivity**: Automates resource configuration, allowing developers to focus on coding rather than managing Kubernetes settings
+- **Optimized Resource Usage**: Ensures resources are neither underutilized nor over-utilized, leading to cost savings
+- **Enhanced Developer Adoption**: Improves the performance of development environments, leading to higher satisfaction and adoption rates amongst developers
+- **Cost Efficiency**: Reduces the overhead associated with manually configuring resources, lowering operational costs
+
+## How it Works
+
+The Resource Manager monitors the actual CPU and memory usage of your pods over time.
+It calculates the average usage and automatically adjusts the resource requests for new pods based on these averages. The system uses the following process:
+
+1. Data Collection: The Resource Manager queries the Kubernetes Metric Server to collect CPU and memory usage data for all pods managed by Okteto
+1. Average Calculation: It calculates the average CPU and memory usage for each container family (grouped by development environment, component, and container name)
+1. Recommendation Generation: The Resource Manager generates new resource requests based on the average usage, applying a correction factor to ensure efficiency
+1. Automatic Application: When a new pod is created, the recommended resource requests are automatically applied unless overridden by specific annotations
+
+## Enabling and Configuring the Resource Manager
+### Enabling the Resource Manager
+The Resource Manager is controlled through the Helm chart configuration. By default, it is enabled with the following settings:
+
+``` yaml
+resourceManager:
+  enabled: true
+  schedule: 300
+  deletePeriod: 15
+  priorityClassName:
+    labels:
+	annotations:
+	resources:
+	recommendations:
+		weight:
+		correction:
+		min:
+			cpu:
+			memory:
+```
+To enable the Resource Manager:
+
+1. Update your Helm values file to include the `resourceManager` block from above
+1. Deploy your Helm chart with the updated configuration
+1. The Resource Manager will start running as a cron job every 300 seconds (5 minutes)
+
+### Enabling via the Okteto Admin UI
+If the Resource Manager is not already enabled via Helm, or if you prefer to manage it from the UI, follow these steps:
+
+1. Log in to the Okteto Admin UI
+1. Navigate to Admin -> General where you can manage system-level settings
+1. Locate the Resource Manager section. If the Resource Manager is installed but not yet enabled, you will see an "Enable/Disable" toggle switch
+1. Toggle the switch to "Enable". This will activate the Resource Manager without needing to manually update the Helm chart
```

Comment:
It's independent. First, the helm configuration to install the RM (default behavior). Then, from the UI you activate it to apply the recommendations